h2/src/main/org/h2/store/Data.java

```diff
@@ -90,6 +92,7 @@
     private static final int LOCAL_DATE = 133;
     private static final int LOCAL_TIMESTAMP = 134;
     private static final byte CUSTOM_DATA_TYPE = (byte)135;
+    private static final int ROW = 27;
```

Comment:
???

Reply:
Its value type is already used in both backend, so an another ID is required.

Reply:
`INTERVAL` data type uses 13 value types and only one storage type.

Reply:
Yeah, a comment would be nice, it just looks like a bug when you see
        case Value.ARRAY:
        case ROW:
in one method and
        case Value.ARRAY:
        case Value.ROW:
in the next method.

Reply:
I placed some comments here in another PR.